src/Dom/Document.php

```diff
@@ -969,6 +978,28 @@ private function get_mustache_tag_placeholders() {
 		return $placeholders;
 	}
 
+	/**
+	 * Covert the emoji AMP symbol (⚡) into pure text.
+	 *
+	 * The emoji symbol gets stripped by DOMDocument::loadHTML().
+	 *
+	 * @param string $source Source HTML string to convert the emoji AMP symbol in.
+	 * @return string Adapted source HTML string.
+	 */
+	private function convert_amp_emoji_attribute( $source ) {
+		return preg_replace( '/(<html [^>]*?)⚡([^\s^>]*)/i', '\1' . self::EMOJI_AMP_ATTRIBUTE . '="\2"', $source, 1 );
```

Comment:
Maybe this is relevant: https://github.com/ampproject/amphtml/issues/25990

Reply:
Resolved via https://github.com/ampproject/amp-wp/pull/4019/commits/6af5a580200b67b1a581502813698ffee1a9feda